src/components/ebay-skeleton/skeleton.stories.js

```diff
@@ -0,0 +1,220 @@
+        },
+        multiline: {
+            control: { type: "boolean" },
+            table: {
+                defaultValue: {
+                    summary: false,
+                },
+            },
+            description:
+                "Boolean flag to make `ebay-skeleton-text` render more than one line",
+        },
+    },
+};
+
+export const Default = Template.bind({});
+Default.args = {
+    style: "width: 220px",
+    renderBody: `<div class="skeleton__textbox" />`,
+};
+Default.parameters = {
+    docs: {
+        source: {
+            code: tagToString("ebay-skeleton", Default.args),
+        },
+    },
+};
+
+export const Avatar = (args) => ({
+    input: args,
+    component: avatarTemplate,
+});
+Avatar.args = {};
+Avatar.parameters = {
+    docs: {
+        source: {
+            code: avatarCode,
+        },
+    },
+};
+
+export const Button = (args) => ({
+    input: args,
+    component: buttonTemplate,
+});
+Button.args = {};
+Button.parameters = {
+    docs: {
+        source: {
+            code: buttonCode,
+        },
+    },
+};
+
+export const ButtonSmall = (args) => ({
+    input: args,
+    component: buttonSmallTemplate,
+});
+ButtonSmall.args = {};
+ButtonSmall.parameters = {
+    docs: {
+        source: {
+            code: buttonSmallCode,
+        },
+    },
+};
+
+export const Text = (args) => ({
+    input: args,
+    component: textTemplate,
+});
+Text.args = {};
+Text.parameters = {
+    docs: {
+        source: {
+            code: textCode,
+        },
+    },
+};
+
+export const TextMultiLine = (args) => ({
+    input: args,
+    component: textMultilineTemplate,
+});
+TextMultiLine.args = {};
+TextMultiLine.parameters = {
+    docs: {
+        source: {
+            code: textMultilineCode,
+        },
+    },
+};
+
+export const TextBox = (args) => ({
+    input: args,
+    component: textboxTemplate,
+});
+TextBox.args = {};
+TextBox.parameters = {
+    docs: {
+        source: {
+            code: textboxCode,
+        },
+    },
+};
+
+export const Image = (args) => ({
+    input: args,
+    component: imageTemplate,
+});
+Image.args = {};
+Image.parameters = {
+    docs: {
+        source: {
+            code: imageCode,
+        },
+    },
+};
+
+export const Tile = (args) => ({
+    input: args,
+    component: tileTemplate,
+});
+Tile.args = {};
+Tile.parameters = {
+    docs: {
+        source: {
+            code: tileCode,
+        },
+    },
+};
+
+export const composite = (args) => ({
+    input: args,
+    component: compositeTemplate,
+});
+composite.args = {};
+composite.parameters = {
+    docs: {
+        source: {
+            code: compositeCode,
+        },
+    },
+};
+
+export const withContent = (args) => ({
+    input: args,
+    component: withContentTemplate,
+});
+withContent.args = {};
+withContent.parameters = {
+    docs: {
+        source: {
+            code: withContentCode,
+        },
+    },
+};
```

Comment:
I have a shortcut way of doing all this. 
https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-button/button.stories.js#L224-L227

The third argument is args if needed. 
https://github.com/eBay/ebayui-core/blob/336467022d6d755e714e48b90e48aefc4d9bca0d/.storybook/utils.js#L24

Reply:
Done.